app/models/deployment.rb

```diff
@@ -36,7 +36,7 @@ def check_stage_ready
       errors.add('stage', 'is not ready to deploy') unless self.stage.deployment_possible?
 
       self.stage.prompt_configurations.each do |conf|
-        errors.add('base', "Please fill out the parameter '#{conf.name}'") unless !prompt_config.blank? && !prompt_config[conf.name.to_sym].blank?
+        errors.add('base', "Please fill out the parameter '#{conf.name}'") unless !prompt_config.blank? && !prompt_config[conf.name].blank?
```

Comment:
The old way of doing things had the configuration parameter names converted into symbols and saved into the configuration hash, now we read them from the db as strings.  This call was using .to_sym to look for a symbol version of a parameter name saved as a string.